source/en/calibration_freshwater.rst

```diff
@@ -0,0 +1,150 @@
+
+* Review the units of your model inputs, and units of observed values, and make sure they're all correct.
+
+* Remember that the InVEST freshwater models are very simple, and may not include processes that have a large impact on water quantity and/or quality in your study area. Review the **How the Model Works** and **Limitations** sections of the model's User Guide chapter for more specific information. It may be that the model is simply not a good match for your study area, in which case you can consider using a different model that is more complex and/or specifically includes the dominant processes in your watershed.
+
+
+.. freshwater_calibration_watersheds:
+
+Delineating watersheds
+======================
+
+When calibrating freshwater models with observed data, we need to delineate the watershed that flows into the point where the observed data gauge is located. Then we can summarize the relevant model result (such as sediment export) within that watershed, and compare that summary with the observed data value. 
+
+Many different tools are available to create watersheds, and you can use whichever one you’re comfortable with. InVEST includes the tool :ref:`DelineateIt <delineateit>` as a simple, effective way of creating watersheds.
+
+Whichever tool you use, they generally require, at a minimum, a digital elevation model (DEM) raster, and a vector (like a shapefile or geopackage) containing the point location(s) to be used as outlets. In this case, the outlet will be the location of a gauge station that collected the observed data. The DEM must be the same one that is used as input to the InVEST freshwater model you’re calibrating.
+
+After running the delineation tool, look at the resulting watershed carefully to make sure that it appears correct. One common problem is that the delineated watershed is very tiny. This is usually caused by the outlet point not being located directly on a stream created by the delineation tool. To fix this, many delineation tools have a “snap” function, where you can specify a distance around the outlet point that the tool should look for a stream, and if one is found within that distance, the tool “snaps” the point to the stream, and delineates the watershed more accurately. If the tool does not have a snap feature, you can manually move the point to lie on the stream network generated by the delineation tool.
+
+Once the watershed is correctly generated, a GIS tool like Zonal Statistics is used to sum the relevant model result raster (such as sediment export or nutrient export) within the watershed. This summarized value is then compared with the observed data value. Alternatively, you can use the generated watershed as an input to the model, which will do the summarizing for you, creating a vector layer whose table contains the summarized values.
+
+Setting up the InVEST Python scripting environment
+==================================================
```

Comment:
Do you think it makes sense for this section to refer to our other docs on installing `natcap.invest` and running python scripts?
* https://invest.readthedocs.io/en/latest/installing.html
* https://invest.readthedocs.io/en/latest/scripting.html#setting-up-your-python-environment

The benefit is that we only have to keep this stuff updated in one place.

Reply:
Yes, great catch Dave. This section was copied from a quick HowTo that we wrote a while back, and I forgot about the readthedocs docs. I've updated that section to point to those two resources.